R/motif_enrichment_HOMER.R

```diff
@@ -98,10 +99,11 @@ dumpJaspar <- function(filename,
     mdb <- utils::getFromNamespace(pkg, ns = pkg)
     if (!methods::existsMethod(f = getMatrixSet, signature = list(x = class(mdb)))) {
         # workaround for JASPAR2024 (no specific TFBSTools::getMatrixSet method yet)
-        if (inherits(mdb, "function") && exists(x = "db", where = paste0("package:", pkg))) {
+        if (inherits(mdb, "function") && exists("db", envir = asNamespace(pkg))) {
             jobj <- do.call(what = mdb, args = list())
-            dbfunc <- get(x = "db", envir = as.environment(paste0("package:", pkg)))
+            dbfunc <- getFromNamespace(x = "db", ns = pkg)
             mdb <- RSQLite::dbConnect(RSQLite::SQLite(), dbfunc(jobj))
```

Comment:
Could add also `SQLite` to `@importFrom RSQLite`

Reply:
Ah, yes, missed that one - thanks!